### DISCUSSION TITLE
Publish artifacts to multiple scms

### DISCUSSION THREAD

**Discussion by timofurrer** [goreleaser/goreleaser#3151] [Ideas, Issue Triage, and General discussions]
I have the situation that I need to publish artifacts both to GitLab and GitHub.

However, this is prevented with [an error](https://goreleaser.com/errors/multiple-tokens/):

>    ⨯ release failed after 0.02s error=gmultiple tokens found, but only one is allowed: GITHUB_TOKEN, GITLAB_TOKEN

Is there a way to achieve that without having to build twice?

**Comment by caarlos0** [comment]
yes, this is not possible at this moment.

Maybe worth writing some sort of "release copier" to copy a release from github to gitlab 🤔

**Reply by timofurrer** [reply]
Is this something that is planned? IMHO, I don't see a reason why there be a technical hurdle :)
(I may actually contribute that ...)

It would need it to sync the other way around, from GitLab to GitHub ... but yeah, it's probably what I'll be doing as a work around.

**Reply by caarlos0** [reply]
there are quite a few places that work differently depending on which scm is being used.

not sure if its worth the refactor, as this is the first time someone asked for it...
